oximeter/oximeter/schema/management-network-data-link.toml

```diff
@@ -0,0 +1,140 @@
+format_version = 1
+
+[target]
+name = "management_network_data_link"
+description = "A network data link related to a management gateway virtual local area network (vlan)"
+authz_scope = "fleet"
+
+versions = [
+    { version = 1, fields = [ "kind", "link_name", "model", "network", "rack_id", "sled_id", "sled_model", "sled_revision", "sled_serial", "switch_id", "switch_model", "switch_revision", "switch_serial", "switch_slot", "asic_fab", "asic_lot", "asic_wafer", "asic_wafer_loc_x", "asic_wafer_loc_y" ] },
+]
+
+[fields.kind]
+type = "string"
+description = "The kind or class of the data link (examples: physical-device, gateway, switch-port)"
+
+[fields.link_name]
+type = "string"
+description = "Name of the data link in the system (examples: oxControlService1, opte0, tfport0)"
+
+[fields.model]
+type = "string"
+description = "Model type of the link, indicating what implements or enables the link itself (examples: TF2, tfportd, STM32H753, opte)"
+
+[fields.network]
+type = "string"
+description = "Network type of the link (examples: primary-data, management, vpc)"
+
+[fields.rack_id]
+type = "uuid"
+description = "ID for the link's rack"
+
+[fields.sled_id]
+type = "uuid"
+description = "ID for the link's sled"
+
+[fields.sled_model]
+type = "string"
+description = "Model number of the link's sled"
+
+[fields.sled_revision]
+type = "u32"
+description = "Revision number of the sled"
+
+[fields.sled_serial]
+type = "string"
+description = "Serial number of the sled"
+
+[fields.switch_id]
+type = "uuid"
+description = "ID of the switch the link is on"
+
+[fields.switch_model]
+type = "string"
+description = "Model number of the switch the link is on"
+
+[fields.switch_revision]
+type = "u32"
+description = "Revision number of the switch the link is on"
+
+[fields.switch_serial]
+type = "string"
+description = "Serial number of the switch the link is on"
+
+[fields.switch_slot]
+type = "u16"
+description = "Slot number of the switch the link is on"
+
+[fields.asic_fab]
+type = "string"
+description = "Fabrication plant identifier of the switch the link is on"
+
+[fields.asic_lot]
+type = "string"
+description = "Lot identifier of the switch the link is on"
+
+[fields.asic_wafer]
+type = "u8"
+description = "Wafer identifier of the switch the link is on"
+
+[fields.asic_wafer_loc_x]
+type = "i16"
+description = "X-coordinate wafer location of the switch the link is on"
+
+[fields.asic_wafer_loc_y]
+type = "i16"
+description = "Y-coordinate wafer location of the switch the link is on"
```

Comment:
I may be misremembering, but I thought we decided that the ASIC-focused fields were not important for the management network. Did I get that wrong?